modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -208,7 +208,7 @@ The low watermark for available memory at process start.
 
 === redpanda_memory_free_memory
 
-Available memory in bytes.
+Total memory potentially available (includes both free and reclaimable memory) to a CPU shard (core), in bytes.
```

Comment:
```suggestion
Total amount of unallocated memory available (includes both free and reclaimable memory) to a CPU shard (core), in bytes.
```

Reply:
@JFlath Better?

Reply:
I think we should pull the word `available` out. It has a specific meaning to us with memory metrics, so using it generically here is a source of confusion. Maybe just:

> Total amount of unallocated memory for a CPU shard (core), in bytes.